conf/airframes/test_boards/aiof7_betafpv.xml

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="AIO F7 Test File">
+
+  <firmware name="fixedwing">
```

Comment:
Do we really want to test this board with a fixedewing firmware ? (same question for the tmotor board)

Reply:
No, these boards will always be used in a quad.